docs/index.rst

```diff
@@ -19,7 +19,13 @@ Get Started
 ===========
 
 - New user? `Register <https://neptune.ai/register>`_ and climb aboard.
-- Registered already? Log in `here <https://neptune.ai/login>`_.
+- Registered already? Log in `here <https://neptune.ai/login>`_, then click **Getting Started** and follow the onboarding instructions:
```

Comment:
All links should be in this format: 

some text |neptune-client-github| some text.

Then at the bottom:
.. |neptune-client-github| raw:: html

    <a href="https://github.com/neptune-ai/neptune-client" target="_blank">Neptune client</a>

Reply:
also links to methods